packages/core/src/Modules/Cashier/Containers/cashier.jsx

```diff
@@ -46,7 +46,7 @@ class Cashier extends React.Component {
                         (this.props.is_p2p_visible && /show_p2p/.test(this.props.location.hash)))
                 ) {
                     options.push({
-                        ...(route.path === routes.cashier_dp2p && { count: this.props.p2p_notification_count }),
+                        ...(route.path === routes.cashier_p2p && { count: this.props.p2p_notification_count }),
```

Comment:
the route in routes-config is still `cashier_dp2p`, right?

Reply:
in the last PR I changed it to cashier_p2p

Reply:
in trader I still see `cashier_dp2p`, can you do a project-wide search to see it and update it?

Reply:
done 👍